recipes/install.rb

```diff
@@ -82,8 +83,7 @@
     sysconfig_file: node['mongodb']['sysconfig_file'][config_type],
     ulimit: node['mongodb']['ulimit'],
     bind_ip: node['mongodb']['config'][config_type]['net']['bindIp'],
-    port: node['mongodb']['config'][config_type]['net']['port'],
-    user: node['mongodb']['user']
+    port: node['mongodb']['config'][config_type]['net']['port']
```

Comment:
Seems like there are a lot of things pulling from `node.mongodb.config[config_type]`, should we extract variable?